tests/conftest.py

```diff
@@ -21,16 +21,21 @@ def shared_setup(fn_isolation):
   pass
 
 
+@pytest.fixture
```

Comment:
@pmdaly I thought this fix was already in master?

Reply:
https://github.com/yearn/yearn-lens/pull/39/files

Reply:
It is already in master, I'm not sure why it's shown here as a change.

Reply:
It should have already been in master from a previous PR of mine.  I'm not sure why it's not in master currently.

Reply:
I'm not sure why it's showing up as a change here.  Possibly master moved prior to my PR? Maybe I need to rebase?